public/theme/boltcms-2021/page.twig

```diff
@@ -39,5 +39,5 @@
 {% endblock main %}
 
 {% block javascripts %}
-    {% include 'partials/_javascripts.twig' %}
+{#    {% include 'partials/_javascripts.twig' %}#}
```

Comment:
If that's not needed, let's remove it :-)

Maybe we don't need the javascripts block at all then?